### ISSUE TITLE
Add support for OTP to New-TssSession

### ISSUE THREAD

**Issue by hawleyOSU** [thycotic-ps/thycotic.secretserver#130]
**Describe the bug**
Users with MFA enabled, specifically Duo, can not authenticate with new-session, and receive an invalid Duo Pin code error. 

**To Reproduce**
1. PowerShell script:

```powershell
 $ses = new-tsssession -SecretServer $url -Credential $cred
```

1. Error/issue observed:

```powershell
Exception: C:\Program Files\WindowsPowerShell\Modules\Thycotic.SecretServer\functions\New-Session.ps1:138:25
Line |
 138 |                          throw $err
     |                          ~~~~~~~~~~
     | {"error":"Invalid Duo pin code."}
```

**Expected behavior**
User should be able to provide an OTP pin or the string "push" as a parameter to new-tsssession. This value should be passed to Invoke-RestApi as $invokeParams.Header. Including ```$Headers = @{"OTP" = $OTP}``` in the header of the oauth2 request will cause Secret Server to issue the request for a push or validate the string as a user OTP. 

**Environment (please complete the following information):**

***Secret Server***
Build/Version: 10.9.0000035

***PowerShell version***

```powershell
Name                           Value
----                           -----
PSVersion                      7.1.3
PSEdition                      Core
GitCommitId                    7.1.3
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by wsmelton** [comment]
TOTP support has not been added to the module as if yet.

**Comment by hawleyOSU** [comment]
It’s not TOTP, it’s Duo HOTP or Push, that’s missing. Any user with Duo can not use the module. If you want to contact me feel free. I looked at the code and it just needs to include the OTP in the token request’s header. I looked at the code and can even show you working code for this. Unfortunately at this time my employer does not allow me to share IP.

Feel free to reclassify this as a feature request if necessary, but until it supports MFA users with Duo can not use the module.

Andy Hawley
